contract_sale/views/abstract_contract_line.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!--FORM view-->
+    <record id="contract_abstract_contract_line_form_view" model="ir.ui.view">
+        <field
+            name="name"
+        >contract.abstract.contract.line form view (in contract_sale)</field>
+        <field name="model">contract.abstract.contract.line</field>
+        <field
+            name="inherit_id"
+            ref="contract.contract_abstract_contract_line_form_view"
+        />
+        <field name="arch" type="xml">
+            <field name="discount" position="attributes">
+                <attribute name="groups">sale.group_discount_per_so_line</attribute>
```

Comment:
This group has changed to module `product` (which should be that way in first place):

```suggestion
                <attribute name="groups">product.group_discount_per_so_line</attribute>
```